apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1306,6 +1384,8 @@ handle_common_event_(cast, {?CHANNEL_CLOSING, ChanId} = Msg, _St, _,
             lager:debug("Channel no longer active: ~p", [Error]),
             close(channel_no_longer_active, D)
     end;
+handle_common_event_({call, From}, {strict_sig_checks, Strict}, _, _, #data{} = D) ->
```

Comment:
Perhaps a comment about why a specific clause is needed here, since the next clause seems to handle all calls (except during `Mode == error_all`)?

Also, check that `Strict` is a boolean.